index.html

```diff
@@ -164,10 +161,10 @@ <h2 class="about-me-title">About me</h2>
       <p class="about-me-text"> I can help you build a product , feature or website Look through some of my work and
         experience! If you like what you see and have a project you need coded,
         don’t hestiate to contact me.</p>
-      <div class="about-button">Get My Resume</div>
+      <div class="about-button"><a href="#">Get My Resume</a></div>
     </div>
     <ul class="abilities">
-      <li class="ability-box">
+      <li class="ability-box language-box">
         <img src="./images/language.png" alt="languages-icon">
         <div class="abi-data">
           <h3>Languages</h3>
```

Comment:
- [ ] [OPTIONAL] Hi You've designed this section perfectly. I however want you to notice that the width of your first `.ability-box` li is a little smaller than the rest. Fixing this will make them align perfectly which will be more in line with the Figma design template.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144516088-34c26a59-7c12-4824-a45e-73d339fb50c3.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/144516150-6ca3c727-7149-497e-b58f-43b6ce8cc9da.png)